app/Support/Rules/CtypeAlnum.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Support\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\InvokableRule;
+use Illuminate\Translation\PotentiallyTranslatedString;
+
+class CtypeAlnum implements InvokableRule
+{
+    /**
+     * @param Closure(string): PotentiallyTranslatedString $fail
+     */
+    public function __invoke($attribute, mixed $value, $fail): void
+    {
+        if (!ctype_alnum($value)) {
+            $fail('validation.ctype_alnum')->translate();
```

Comment:
Rereading the changes I think it's not necessary to add a custom validation message; reuse `validation.alpha_num` instead?

Reply:
Makes sense. Unless you want to expand on the message:
`The :attribute must contain only unaccented letters and numbers`

Might also want to include a comment indicating that this differs from the built-in `alpha_num` validation in that it restricts the input to ASCII characters.